src/editor/WelcomeDialog.cpp

```diff
@@ -0,0 +1,39 @@
+#include "WelcomeDialog.h"
+#include "ui_WelcomeDialog.h"
+#include "Settings.h"
+#include <QStyle>
+#include <QIcon>
+#include <QPainter>
+#include <QToolTip>
+
+WelcomeDialog::WelcomeDialog(QWidget *parent) :
+  QDialog(parent),
+  ui(new Ui::WelcomeDialog)
+{
+  ui->setupUi(this);
+  ui->labelVersion->setText(qApp->applicationName() + " " + qApp->applicationVersion());
+  ui->showAgainCheck->setChecked(Settings::ShowLandingPage::get());     //Shouldn't be necessary but if it is somehow false i want the checkbox to be right
+
+  QString connectToolTip =      tr("Connect to an existing server.");
+  QString newToolTip =          tr("Create a new project without hosting.");
+  QString openToolTip =         tr("Open an existing project for offline use.");
+
+  ui->buttonConnect->setToolTip(connectToolTip);
+  ui->buttonNew->setToolTip(newToolTip);
+  ui->buttonOpen->setToolTip(openToolTip);
+
+  connect(ui->buttonNew, SIGNAL(released()), this, SLOT(buttonNewPressed()));
```

Comment:
Nit, is it okay if you use the function pointer syntax as in the other places here and below? E.g. &QButton::released instead of SIGNAL(...), &WelcomeDialog::buttonNewPressed instead of SLOT(...)

Reply:
done